backends/tfhe-cuda-backend/cuda/include/integer/integer_utilities.h

```diff
@@ -3079,7 +3079,7 @@ template <typename Torus> struct int_are_all_block_true_buffer {
                                            streams[0], gpu_indexes[0]);
       is_max_value =
           new int_radix_lut<Torus>(streams, gpu_indexes, gpu_count, params, 2,
-                                   num_radix_blocks, allocate_gpu_memory);
+                                   max_value, allocate_gpu_memory);
```

Comment:
shouldn't this be num_chunks instead?

Reply:
Oh  you're right it should be `max_chunks` here

Reply:
We apply the lut with this:
```
      integer_radix_apply_univariate_lookup_table_kb<Torus>(
          streams, gpu_indexes, gpu_count, tmp_out, accumulator, bsks, ksks,
          num_chunks, lut);
```
so num_chunks is the amount of inputs, and we have to allocate with the maximum possible num_chunks vaue, which is `max_chunks`